src/main/java/dev/blachut/svelte/lang/Svelte.flex

```diff
@@ -73,7 +77,7 @@ WHITE_SPACE=\s+
   ")"                { leftParenCount -= 1; if (leftParenCount == 0) { return END_PAREN; } else { return CODE_FRAGMENT; } }
 }
 
-<SVELTE_INTERPOLATION, SVELTE_TAG, SVELTE_TAG_PAREN_AWARE> {
+<SVELTE_INTERPOLATION, SVELTE_TAG, SVELTE_ATTR, SVELTE_TAG_PAREN_AWARE> {
```

Comment:
@unlocomqx - I don't think that switching to `YYINITIAL` for SVELTE_ATTR/"}" is a good idea, because we are actually inside the tag or string, and we need to return to either HTML_TAG or TAG_STRING state, otherwise we cannot detect that tag or string are really closed. You can test it with `<Hello {value} />`. You will see that parser will complain on "Element Hello is not closed".